packages/react-sdk/src/components/ImportWhiteboardDialog/ImportWhiteboardDialog.tsx

```diff
@@ -172,11 +252,120 @@ export function ImportWhiteboardDialog({
         <Button
           onClick={handleOnImport}
           variant="contained"
-          disabled={importedWhiteboard?.isError}
+          disabled={error || loading}
         >
           {t('boardBar.importWhiteboardDialog.success.import', 'Import')}
         </Button>
       </DialogActions>
     </Dialog>
   );
 }
+
+function readNWB(file: File): Promise<{
```

Comment:
Puh - can we put this to somewhere else? Even if you use it only from this component maybe put it into a separate file next to it. The component file gets a little bit crowded.